apps/mark/frontend/src/pages/review_page.test.tsx

```diff
@@ -20,7 +23,7 @@ it('Renders ContestPage', () => {
   screen.getByText(/color & size/i);
 });
 
-it('Renders ContestPage in Landscape orientation', () => {
+it('Renders ReviewPage in Landscape orientation', () => {
```

Comment:
Where are we w.r.t. supporting or not supporting landscape? Is this test necessary anymore?

Reply:
I think we'll be able to move support eventually, but Caroline mentioned we'll support it in the short term (though I'm already blanking out on the specifics of the conversation). Will follow up again with her and Roe on what  the timeline for that looks like - would simplify a lot of code if we could remove it altogether